core/src/test/scala/akka/persistence/cassandra/CassandraLifecycle.scala

```diff
@@ -119,8 +120,8 @@ trait CassandraLifecycle extends BeforeAndAfterAll with TestKitBase {
   }
 
   override protected def afterAll(): Unit = {
-    shutdown(system, verifySystemShutdown = true)
     externalCassandraCleanup()
+    shutdown(system, verifySystemShutdown = true)
```

Comment:
this was another test change; cleanup before shutdown

Reply:
this will mean that the keyspace will get deleted while the journal is still running, not sure that's a problem

Reply:
I think the reason I changed this was that the cleanup is now using the `session` and then the ActorSystem must still be running.